main.py

```diff
@@ -78,5 +80,67 @@ def calorie_consumption_calculator(food_type: str, food_mass: int, food_data: di
     return quantity_of_eaten_calories
 
 
+def aerobic_exercise_calories_burning(user_weight: int, exercise_name: str, exercise_time: float,
+                                      exercise_intensity: int = 3) -> float:
+    """
+    Function to calculate burned calories after aerobic type of exercise
+    :param exercise_name: name of exercise (str)
+    :param exercise_time: exercise duration (float)
+    :param exercise_intensity: approximate level of intensity (default 3) (int)
+    :param user_weight: weight of user (int)
+    :return: number of calories (float)
+    """
+    result = None
+    connection, crsr = psql_connection("database.ini", "postgresql")
+    exercise_list = crsr.execute("SELECT exercise_name FROM exercises")
+    exercise_list = crsr.fetchall()
+    exercise_list = [exercise_name[0] for exercise_name in exercise_list]
+
+
+    if exercise_name in exercise_list:
+        exercise_met_average = crsr.execute(f"SELECT exercise_met_average FROM exercises "
+                                            f"WHERE exercise_name = '{exercise_name}'")
+
+        exercise_met_average = crsr.fetchall()[0][0]
+        # print(exercise_met_average)
+        result = (exercise_met_average * 3.5 * user_weight * exercise_time * (exercise_intensity / 3)) / 200
+    else:
+        print("Exercise not in db")
+
+    if crsr is not None:
+        crsr.close()
+    if connection is not None:
+        connection.close()
+        print("Database connection terminated.")
+
+
+    return result
+
+# print(aerobic_exercise_calories_burning(76, "jum", 60, 3))
+
+def add_new_user(first_name, last_name, gender, age, height, weight):
+    user = User(first_name, last_name, gender, age, height, weight)
+
+    connection, crsr = psql_connection("database.ini", "postgresql")
```

Comment:
I don't know how you set the table, but normally the id should be generated automatically (with primary key), so no need to check the last id and insert it